op-challenger/game/fault/types/position.go

```diff
@@ -146,24 +133,12 @@ func (p Position) ToGIndex() *big.Int {
 
 // bigMSB returns the index of the most significant bit
 func bigMSB(x *big.Int) int {
-	if x.Cmp(common.Big0) == 0 {
```

Comment:
what's the issue with using `common.Big0` here?

Reply:
It failed to ever be "0" - that is, tests would just hang in the below loop. I'm not sure why this is - maybe something is updating the value this points to behind the scenes causing it to not be "0". But since `new(big.Int)` is logical, we should just go with that to avoid any errors.

Reply:
Yea, it's possible that someone is accidentally writing to the common.Big*. That's actually pretty concerning because it could be causing issues with other parts of our system.

Reply:
Yep, silent type errors scare me